default/python/AI/AIDependencies.py

```diff
@@ -399,6 +406,22 @@
     "SHP_SOLAR_CONT",
 )
 
+
+# Industry modifiers per population - [[EARLY_PRIORITY]]: Affected by species modifiers
+INDUSTRY_EFFECTS_PER_POP_MODIFIED_BY_SPECIES = {
```

Comment:
I have feeling that this should be replaced by function without arguments. (tag extraction can be done inside)